### ISSUE TITLE
DS2 small model WER 

### ISSUE THREAD

**Issue by inchpunch** [NVIDIA/OpenSeq2Seq#386]
I ran DS2 small model in evaluation mode without using language model. WER is around 29%. I saw in the default base_params num_epochs is 12. Is this performance expected? Thanks!

**Comment by borisgin** [comment]
What data set have you used for evaluation?

**Comment by inchpunch** [comment]
I used librivox-dev-clean, the default dataset in the example_config for small model.

**Comment by wd929** [comment]
I got 32% after training. Maybe it does not have any problem due to the small network and dataset compared with the original DS2 and Librispeech.

**Comment by inchpunch** [comment]
It looks so. BTW that was from default "num_epochs"=12. I ran 28 epochs and then got 25.2%.
